### DISCUSSION TITLE
Kyuubi + Spark + Dora metrics question.

### DISCUSSION THREAD

**Discussion by yikf** [apache/kyuubi#4654] [Q&A] [answered]
I'm trying to implement something very similar to Dora metrics in our Kyuubi environment. I wanted to ask one question:

Is it possible with the existing metrics about Spark processes run with Kyuubi, to get two metrics one of which tells the number of crashed processes due to a system resource problem and the other one tells Total the number of crashes?

**Comment by yikf** [comment]
cc @pan3793 @turboFei ; 

This was a question from an Apache kyuubi user @rachlenko,  and I reposted it.

I'm under the impression that there are no existing metrics to capture these two things.

Do you have any ideas?

**Comment by turboFei** [comment, marked as answer]
> Is it possible with the existing metrics about Spark processes run with Kyuubi,


now there is no metrics for this, because the engine is decoupled with server.
